docs/manual/java/guide/cluster/Cluster.md

```diff
@@ -34,18 +34,45 @@ You could imagine using cluster features across different services, but we recom
 
 ## Joining
 
-A service instance joins a cluster when the service starts up. In development you are typically only running the service on one cluster node. No explicit joining is necessary; the [[Lagom Development Environment|DevEnvironment]] handles it automatically. In production, you need to implement the joining yourself as follows.
+A service instance joins a cluster when the service starts up.
 
-First, define some initial contact points of the cluster, so-called seed nodes. You can define seed nodes in `application.conf`:
+### Joining during development
+In development you are typically only running the service on one cluster node. No explicit joining is necessary; the [[Lagom Development Environment|DevEnvironment]] handles it automatically.
 
-    akka.cluster.seed-nodes = [
-      "akka.tcp://MyService@host1:2552",
-      "akka.tcp://MyService@host2:2552"]
+
+### Joining during production
+
+Starting from version 1.5.0, Lagom offers support for [Akka Cluster Bootstrap](https://developer.lightbend.com/docs/akka-management/current/bootstrap/). Akka Cluster Bootstrap is enabled by default in production mode and disabled in development and test mode.
+
+Akka Cluster Bootstrap helps forming (or joining to) a cluster by using [Akka Discovery](https://doc.akka.io/docs/akka/current/discovery/index.html) to discover peer nodes. It is an alternative to configuring static seed-nodes in dynamic deployment environments such as on Kubernetes or AWS.
+
+It builds on the flexibility of Akka Discovery, leveraging a range of discovery mechanisms depending on the environment you want to run your cluster in.
+
```

Comment:
We still need to document here the minimal akka discovery settings needed for production. 

I don't know yet how this should look like. I don't want to repeat Akka Cluster Bootstrap and Akka Discovery documentation, neither should we be too specific and explain how to use it with Kubernetes, for instance. 

I prefer to have it minimal for now and add more details as we move forward and document the health-check and akka management.

Reply:
Make sense to me. Let's then edit the health-check issue to include in its scope that we will add proper documentation here. Or maybe have another issue. WDYT?